cfn-model.gemspec

```diff
@@ -9,6 +9,13 @@ Gem::Specification.new do |s|
   s.summary       = 'cfn-model'
   s.description   = 'An object model for CloudFormation templates'
   s.homepage      = 'https://github.com/stelligent/cfn-model'
+  s.metadata      = {
+    'bug_tracker_uri'   => "#{s.homepage}/issues",
+    'changelog_uri'     => "#{s.homepage}/releases",
+    'documentation_uri' => "https://www.rubydoc.info/gems/#{s.name}/#{s.version}",
```

Comment:
```suggestion
```

I don't think this will be a valid document link for this project. I think it should be fine without it.

Reply:
I just copied what was already on the Rubygems page: https://rubygems.org/gems/cfn-model
See the 'Documentation' link in the 'Links' section.

FYI: it links to https://www.rubydoc.info/gems/cfn-model/0.6.0

To change it on that page, we'll need to provide a value for `documentation_uri` here. Do you have an alternative? Perhaps the readme file in the repo: https://github.com/stelligent/cfn-model/blob/HEAD/README.md

Reply:
Ah, okay. Makes sense.